src/styles/mixins/selectors.scss

```diff
@@ -0,0 +1,6 @@
+@mixin hover-fade {
+  transition: 0.4s ease-out;
```

Comment:
We talked about this on a call and decided to back this off a bit (make it a little faster), and in a subsequent PR as it is applied to one of the basic button mixins.